### PR TITLE
Release/6.12.0

### PR DESCRIPTION
**Pull request by ethan605**
# Fixed

- UI: Fix camera view not lining up with Document Live Capture overlay.
- Public: Fix file selector "capture" prop for WebSDK inside iOS WebView
- Public: Fix `CROSS_DEVICE_START` user analytic event for integrators never being dispatched when user switches to the Cross Device flow
- UI: Update copy in Face Liveness Video intro screen from 25s to 20s to reflect the correct time limit
- Remove old locale key type definitions that are no longer used/exist in code base.

# Changed

- UI: Accessibility - Make "Tips" heading in Cross Device SMS Sent and Mobile Connected screens a level 3 heading.
- UI: Accessibility - Change Cross Device Send Link alternate option text to be an ARIA heading level 2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,20 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
-## [next-version]
+## [6.12.0] - 2021-08-09
```
Comment (kopijunkie) [inline CHANGELOG.md:9]: Need to update the date since we're aiming to release on 10 Aug now.

File: src/components/Camera/style.scss [hunk 2]
```diff
@@ -117,8 +119,20 @@
...
+
+    @media (orientation: landscape) {
+      height: auto;
+      width: 100%;
+    }
+
+    @media (orientation: portrait) {
+      height: auto;
+      width: 100%;
+    }
```
Comment (kopijunkie) [inline src/components/Camera/style.scss:135]: Just realised this is duplicated & unnecessarily in separate media queries 🙈

**Review by kopijunkie — CHANGES_REQUESTED** [review]
Small change in CHANGELOG (release date). Other thing is more a minor code clean up.

File: src/components/Camera/style.scss (hunk 2 above)
Comment (ethan605) [reply src/components/Camera/style.scss]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (ethan605) [reply CHANGELOG.md]: Updated

**Review by kopijunkie — APPROVED** [review]

**Pull request merged**